components/results_table/wikis/starcraft2/results_table_custom.lua

```diff
@@ -0,0 +1,280 @@
+local AwardsTable = Lua.import('Module:ResultsTable/Award', {requireDevIfEnabled = true})
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local UNDEFINED_TIER = 'undefined'
+local ALL_KILL_ICON = '[[File:AllKillIcon.png|link=All-Kill Format]]'
+local DEFAULT_EVENT_ICON = ''
+local TBD = 'TBD'
+local MAXIMUM_NUMBER_OF_PLAYERS_IN_PLACEMENTS = 20
+
+local CustomResultsTable = {}
+
+-- Template entry point
+function CustomResultsTable.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.playerLimit = MAXIMUM_NUMBER_OF_PLAYERS_IN_PLACEMENTS
+
+	if Logic.readBool(args.awards) then
+		return CustomResultsTable.awards(args)
+	end
+
+	args.data = Json.parseIfTable(Variables.varDefault('achievements'))
+	args.manualContent = args[1]
+
+	local resultsTable = ResultsTable(args)
+
+	-- overwrite functions
+	resultsTable.tierDisplay = CustomResultsTable.tierDisplay
+	resultsTable.rowHighlight = CustomResultsTable.rowHighlight
+	resultsTable.processLegacyVsData = CustomResultsTable.processLegacyVsData
+
+	local buildTable = resultsTable:create():build()
+
+	CustomResultsTable._storeAllKill()
+
+	return buildTable
+end
+
+function CustomResultsTable.awards(args)
+	args.data = Json.parseIfTable(Variables.varDefault('awardAchievements'))
+
+	if Logic.readBool(args.achievements) and Table.isEmpty(args.data) then
+		return
+	end
+
+	local awardsTable = AwardsTable(args)
+
+	-- overwrite functions
+	awardsTable.tierDisplay = CustomResultsTable.tierDisplay
+	awardsTable.rowHighlight = CustomResultsTable.rowHighlight
+
+	local sectionHeader = ''
+	if Logic.readBool(args.achievements) then
+		sectionHeader = '\n=====Notable Awards=====\n'
+	end
+
+	return sectionHeader .. tostring(awardsTable:create():build())
+end
+
+function CustomResultsTable:tierDisplay(placement)
+	local tierDisplay = Tier.text.tiers[placement.liquipediatier] or UNDEFINED_TIER
+
+	tierDisplay = Page.makeInternalLink(
+		{},
+		tierDisplay,
+		tierDisplay .. ' Tournaments'
+	)
+
+	local tierTypeDisplay = Tier.text.typesShort[(placement.liquipediatiertype or ''):lower()]
+
+	local sortValue = placement.liquipediatier .. (tierTypeDisplay or '')
+
+	if not tierTypeDisplay then
+		return tierDisplay, sortValue
+	end
+
+	return tierDisplay .. ' (' .. tierTypeDisplay .. ')', sortValue
+end
+
+function CustomResultsTable:rowHighlight(placement)
+	if Logic.readBool(placement.publishertier) then
+		return 'sc2premier-highlighted'
+	end
+end
+
+function CustomResultsTable:processLegacyVsData(placement)
+	if Table.isEmpty(placement.lastvsdata) then
+		local opponent = (placement.extradata or {}).vsOpponent or {}
+		placement.lastvsdata = Table.merge(
+			Opponent.toLpdbStruct(opponent or {}),
+			{groupscore = placement.groupscore, score = placement.lastvsscore}
+		)
+	end
+
+	return placement
+end
+
+function CustomResultsTable.allKillRow(frame)
```

Comment:
```suggestion
-- All Kill are manual tables that blablabla
function CustomResultsTable.allKillRow(frame)
```
Add comment to explain all kill. Based on the code I assume all input is manual. Though also a question is why does it exist?